contracts/ERC721M.sol

```diff
@@ -110,13 +102,6 @@ contract ERC721M is IERC721M, ERC721AQueryable, Ownable, ReentrancyGuard {
         _;
     }
 
-    /**
-     * @dev Returns cosigner address.
-     */
-    function getCosigner() external view override returns (address) {
```

Comment:
Do these need to be removed from the main `ERC721M` contract? Could we just remove them from IERC721M but keep them in the actual contract?

Reply:
It's not quite useful and ME has no references. We don't have any use case that could change cosigner. And we don't have issues historically to mis-set cosigner. Maybe good to keep them for debugging convenience but I think it's easier to just reset `cosigner`.